app/src/examples/LayoutAnimations/NestedNativeStacksWithLayout.tsx

```diff
@@ -3,10 +3,15 @@ import { Button, StyleSheet, View } from 'react-native';
 
 import React from 'react';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { useNavigationBuilder } from '@react-navigation/native';
 
 const Stack = createNativeStackNavigator();
 
-function First({ navigation }: { navigation: any }) {
+function First({
+  navigation,
+}: {
+  navigation: ReturnType<typeof useNavigationBuilder>['navigation'];
+}) {
```

Comment:
There must be a better way to do this, maybe we need to pass some type to `createNativeStackNavigator`?

Reply:
You can see that I've started from here and it was before I've learned how StackNavigator works 😆

Reply:
[08842e9](https://github.com/software-mansion/react-native-reanimated/pull/5361/commits/08842e9ecd7250acf96afc002d3bd9ec388fa950)